### DISCUSSION TITLE
repocache.securityonion.net

### DISCUSSION THREAD

**Discussion by mobtechsec1** [Security-Onion-Solutions/securityonion#9245] [Unsupported Versions]
repo.securityonion.net is down?
![image](https://user-images.githubusercontent.com/119471459/204745957-dcee2e4d-d4e0-4ce3-b955-c26bf83e7a40.png)

**Comment by dougburks** [comment]
The title references `repocache.securityonion.net` but the rest references `repo.securityonion.net`. Which are you asking about?

repocache.securityonion.net doesn't actually exist on the Internet. It's a special URL used by nodes in a distributed deployment to pull from the repo on the manager.

repo.securityonion.net isn't accessible via HTTP so http://repo.securityonion.net is always going to show the Cloudflare error `Web server is down`. A better test would be to request an actual file from repo.securityonion.net via HTTPS. For example:
```
curl -v https://repo.securityonion.net/file/securityonion-repo/securityonion/repodata/repomd.xml
```
